lib/playwright/locator_impl.rb

```diff
@@ -488,5 +488,22 @@ def all_text_contents
     def highlight
       @frame.highlight(@selector)
     end
+
+    def expect(expression, options)
+      if options.key? :expectedValue
+        options[:expectedValue] = JavaScript::ValueSerializer
+          .new(options[:expectedValue])
+          .serialize
+      end
+
+      @frame.channel.send_message_to_server_result(
+        "expect",
+        {
+          selector: @selector,
+          expression:,
```

Comment:
```suggestion
          expression: expression,
```

For Ruby 2.7 compatibility, we have to explicitly specify the value.

```
SyntaxError:
  /home/runner/work/playwright-ruby-client/playwright-ruby-client/lib/playwright/locator_impl.rb:503: syntax error, unexpected ','
            expression:,
                       ^
```